src/test/components/CallTreeSidebar.test.js

```diff
@@ -18,11 +21,11 @@ import type { CallNodePath } from '../../types/profile-derived';
 describe('CallTreeSidebar', function() {
   function setup() {
     const { profile, funcNamesDictPerThread } = getProfileFromTextSamples(`
-      A A A
-      B B B
-      C C H
-      D F I
-      E E
+      A    A    A A
+      B    B    B B
+      Cjs  Cjs  H H
+      D    F    I
+      Ejs  Ejs
```

Comment:
ending the function names with "js" makes `getProfileFromTextSamples` generate JS stacks.